solidity/dashboard/src/css/typography.less

```diff
@@ -17,71 +17,81 @@ body {
   font-family: 'Work Sans', sans-serif;
   font-weight: normal;
   font-size: 1rem;
+  line-height: 1.5rem;
   color: @color-black;
-  line-height: 1.44;
+  font-feature-settings: 'tnum' on, 'lnum' on, 'zero' on;
 
   @media @tablet-up {
     line-height: 1.79;
   }
 }
 
+h1, .h1,
+h2, .h2,
+h3, .h3,
+h4, .h4,
+h5, .h5,
+h6, .h6 {
+  font-family: 'Work Sans', sans-serif;
+  font-feature-settings: 'tnum' on, 'lnum' on, 'zero' on;
+  margin: 0;
+}
+
 h1, .h1 {
-  font-family: 'Vitesse', sans-serif;
+  font-weight: 800;
+  font-size: 3rem;
+  line-height: 3.25rem;
+  letter-spacing: -2px;
+}
+
+.h1--alt {
```

Comment:
Wondering if we have any way of expressing _why_ there's an `alt` here, i.e. name it in a more semantic way, or if this makes sense more as a class that can be mixed in separately. Would require a lot of reworking, so I don't think it's worth any time right now---but wanted to see if we could be thinking about that.

Reply:
Hmm based on the KEEP Design System `alt(alternative)` makes sense only with h1 and h2 tag. Out of idea which tag would fit better here 🤔